src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -3,8 +3,6 @@ import firebase from "firebase/app";
 import * as Yup from "yup";
 
 import {
-  GIF_RESIZER_URL,
```

Comment:
Check if this is being used anywhere else, if it's not, you can remove it from settings.ts

Reply:
It's used in the old admin `DetailsForm` component's validation schema.